.github/workflows/build-coq-bignums.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    # the OS must be GNU/Linux to be able to use the docker-coq-action
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - 'coqorg/coq:dev'
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+      - uses: coq-community/docker-coq-action@v1
+        with:
+          opam_file: 'coq-bignums.opam'
```

Comment:
@proux01 thanks for opening this PR.
The workflow needs to be extended to run the same tests as the Travis CI config:
```suggestion
      - uses: coq-community/docker-coq-action@v1
        with:
          opam_file: 'coq-bignums.opam'
          custom_image: ${{ matrix.image }}
          custom_script: |
            startGroup Print opam config
              opam config list; opam repo list; opam list
            endGroup
            startGroup Build dependencies
              opam pin add -n -y -k path $PACKAGE $WORKDIR
              opam update -y
              opam install -y -j 2 $PACKAGE --deps-only
            endGroup
            startGroup Build
              opam install -y -v -j 2 $PACKAGE
              opam list
            endGroup
            startGroup Workaround permission issue
              sudo chown coq:coq .
            endGroup
            startGroup Run tests
              make -C tests
            endGroup
            startGroup Uninstallation test
              opam remove $PACKAGE
            endGroup
      - name: Revert permissions
        if: ${{ always() }}
        run: sudo chown 1001:116 .
```
Admittedly, this is a bit verbose (albeit less than with Travis :), but we'll be able to refactor/simplify this as soon as https://github.com/coq-community/docker-coq-action/issues/22#issuecomment-695803544 is implemented!

Reply:
Thanks, I wouldn't have found this myself.